### ISSUE TITLE
[Testing Party] Unable to add location and select markers

### ISSUE THREAD

**Issue by suryasdas** [gcivil-nyu-org/INET-Monday-Spring2023-Team-1#222]
When I try to create a new event through the verified account credentials, I am not able to add the location for the event. I am also not able to see any available markers for the event.

**Comment by aaqilkhoja** [comment]
Are you using Safari?

**Comment by suryasdas** [comment]
Yes I tried this on safari.Thank you,Surya DasOn Apr 15, 2023, at 2:01 PM, AaqilKhoja ***@***.***> wrote:﻿
Are you using Safari?
